modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -235,7 +248,7 @@ information for the bonded interface.
 [source,terminal]
 ----
 bond=bond0:em1,em2:mode=active-backup
-ip=bond0:dhcp
+fail_over_mac=1
```

Comment:
This is an unsupported value for fail_over_mac.

Also, this won't work as written. It needs to be part of the colon-separated list on the bond= line. That said, the default value is 0, which is what we recommend anyway, so I'm not sure we even need to explicitly include this option.